packages/http-client-csharp/package.json

```diff
@@ -47,6 +47,8 @@
   },
   "peerDependencies": {
     "@azure-tools/typespec-azure-core": ">=0.36.0 <1.0.0",
+    "@azure-tools/typespec-azure-resource-manager": ">=0.36.0 <1.0.0",
```

Comment:
I don't think we want these dependencies here.  The `@azure-tools/typespec-azure-core` will also be going away here soon.

These dependencies will exist in the azure plugin for the emitter.

Reply:
sounds good. I'll close this PR.